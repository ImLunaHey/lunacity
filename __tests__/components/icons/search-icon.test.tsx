import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { SearchIcon } from '@app/icons/search-icon';

describe('SearchIcon', () => {
    it('renders', () => {
        const { container } = render(<SearchIcon />);
        expect(container).toMatchSnapshot();
    });

    it('renders with custom size', () => {
        const { container } = render(<SearchIcon size={200} />);
        expect(container).toMatchSnapshot();
    });

    it('renders with custom width/height', () => {
        const { container } = render(<SearchIcon width={10} height={10} />);
        expect(container).toMatchSnapshot();
    });
});
