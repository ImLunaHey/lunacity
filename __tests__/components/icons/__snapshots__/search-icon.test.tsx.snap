// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchIcon renders 1`] = `
<div>
  <svg
    fill="none"
    height="24"
    viewBox="0 0 24 24"
    width="24"
  >
    <path
      d="M11.5 21a9.5 9.5 0 1 0 0-19 9.5 9.5 0 0 0 0 19ZM22 22l-2-2"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2"
    />
  </svg>
</div>
`;

exports[`SearchIcon renders with custom size 1`] = `
<div>
  <svg
    fill="none"
    height="200"
    viewBox="0 0 24 24"
    width="200"
  >
    <path
      d="M11.5 21a9.5 9.5 0 1 0 0-19 9.5 9.5 0 0 0 0 19ZM22 22l-2-2"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2"
    />
  </svg>
</div>
`;

exports[`SearchIcon renders with custom width/height 1`] = `
<div>
  <svg
    fill="none"
    height="10"
    viewBox="0 0 24 24"
    width="10"
  >
    <path
      d="M11.5 21a9.5 9.5 0 1 0 0-19 9.5 9.5 0 0 0 0 19ZM22 22l-2-2"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2"
    />
  </svg>
</div>
`;
